app/views/locations/new.erb

```diff
@@ -18,7 +18,14 @@
   <div class='govuk-grid-column-two-thirds'>
     <%= link_to 'Back to list', ips_path, class: 'govuk-back-link' %>
 
-    <h2 class='govuk-heading-l'>Add a new location</h2>
+    <% if current_organisation.locations.empty? %>
+      <h2 class='govuk-heading-l'>Add a new location</h2>
+      <div class="govuk-inset-text">
+        <strong>You have no existing locations to add a new IP address to.</strong>
```

Comment:
I'm trying to work out what confusion this message deals with, or if it's just as clear without it.

I considered:
```suggestion
        <strong>You must add a location with your first IPs.</strong>
```
but this might suggest to some users that they can't use this form yet, and need to find some "new location" in the app before they're allowed to use it.

Then there's just the straight re-word of what you have:
```suggestion
        <strong>You have no locations to which you can add a new IP.</strong>
```
but I think this has the same problem as suggestion no.1, users might think they're not allowed to use this form yet.

I would vote for removing this text, and adding something back in if user research shows users are confused when they land here from "Add IP address".